cpu/ezr32wg/Makefile

```diff
@@ -4,4 +4,7 @@ MODULE = cpu
 # add a list of subdirectories, that should also be build
 DIRS = periph $(RIOTCPU)/cortexm_common
 
+#
```

Comment:
I use empty comment lines to logically seperate lines from the comment above. Do you think it is necessary to add "ignore file for link time optimization" is necessary for all these?